localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
+d_spider_philosophy;The Spider Council;Le Conseil-Araignée;Rat der Spinne;;El Consejo de la Araña;;;;;;;;;x
+d_spider_philosophy_adj;Council;Conseil;rätisch;;Consejo;;;;;;;;;x
+d_water_deities;The Church of the Tides;L'Église des Marées;Die Kirche der Gezeiten;;Iglesia de las Mareas;;;;;;;;;x
+d_water_deities_adj;Tide;Marée;gezeitisch;;Marea;;;;;;;;;x
+d_cult_of_the_candle;The Church of the Candle;Le Culte de la Bougie;Die Kirche der Kerze;;La Iglesia de la Vela;;;;;;;;;x
+d_cult_of_the_candle_adj;Wax;Cire;wächsern;;Cera;;;;;;;;;x
+d_agamaggan_worship;The Cult of Agamaggan;Le Culte d'Agamaggan;Kult von Agamaggan;;El culto de Agamaggan;;;;;;;;;x
 d_agamaggan_worship_adj;Agamaggan's;;agamagganisch;;;;;;;;;;;x
-d_theradras_worship;The Cult of Theradras;Le Culte de Theradras;Kult von Theradras;;;;;;;;;;;x
+d_theradras_worship;The Cult of Theradras;Le Culte de Theradras;Kult von Theradras;;El culto de Theradras;;;;;;;;;x
 d_theradras_worship_adj;Theradras';;theradrasisch;;;;;;;;;;;x
-d_primitive_shamanism;The Sacred Woods;;Die heiligen W�lder;;;;;;;;;;;x
-d_primitive_shamanism_adj;Shamanic;;schamanisch;;;;;;;;;;;x
-d_orcish_shamanism;The Farseers;Les Longs-voyants;Die Scharfseher;;;;;;;;;;;x
-d_orcish_shamanism_adj;Shamanic;;schamanisch;;;;;;;;;;;x
-d_shamanism;The Shamanic Council;Le Conseil chamanique;Der schamanische Rat;;;;;;;;;;;x
-d_shamanism_adj;Shamanic;;schamanisch;;;;;;;;;;;x
-d_earth_mother_worship;The Earth-Temple;Le Temple de la Terre;Tempel der Erde;;;;;;;;;;;x
-d_earth_mother_worship_adj;Earthen;;irdisch;;;;;;;;;;;x
-d_tidemother;The Tidesages;;Die Gezeitenweisen;;;;;;;;;;;x
-d_tidemother_adj;Tidesage;;;;;;;;;;;;;x
-d_odyn;The Halls of All-Father;;Die Hallen des Allvaters;;;;;;;;;;;x
-d_odyn_adj;Odyn's;;odynisch;;;;;;;;;;;x
-d_helya;The Temple of the Sea-Witch;Le Temple de la Sorci�re des mers;Tempel der Seehexe;;;;;;;;;;;x
-d_helya_adj;Helya's;;helyasisch;;;;;;;;;;;x
+d_primitive_shamanism;The Sacred Woods;Les Bois Sacrés;Die heiligen Wälder;;El bosque Sagrado;;;;;;;;;x
+d_primitive_shamanism_adj;Shamanic;Chamanique;schamanisch;;Chamánico;;;;;;;;;x
+d_orcish_shamanism;The Farseers;Les Longs-voyants;Die Scharfseher;;Los Clarividentes;;;;;;;;;x
+d_orcish_shamanism_adj;Shamanic;Chamanique;schamanisch;;Chamánico;;;;;;;;;x
+d_shamanism;The Shamanic Council;Le Conseil chamanique;Der schamanische Rat;;El Consejo Chamánico;;;;;;;;;x
+d_shamanism_adj;Shamanic;Chamanique;schamanisch;;Chamánico;;;;;;;;;x
+d_earth_mother_worship;The Earth-Temple;Le Temple de la Terre;Tempel der Erde;;El Templo de la Tierra;;;;;;;;;x
+d_earth_mother_worship_adj;Earthen;En Terre;irdisch;;De barro;;;;;;;;;x
+d_tidemother;The Tidesages;Des Eaugures;Der Gezeitenweisen;;Los Sabiomar;;;;;;;;;x
+d_tidemother_adj;Tidesage;Eaugure;gezeitenweise;;Sabiomar;;;;;;;;;x
+d_odyn;The High Halls of the All-Father;Les salles exaltées du Père-tout-puissant;Den heiligen Hallen des allmächtigen Vaters;;La Cámara Alta del Gran Padre;;;;;;;;;x

```

Comment:
Die Hohen Hallen des Allvaters

Reply:
This is actually a direct copy-paste from official lore text, are you sure we should change it?
https://de.wowhead.com/quest=38331/havis-test

```
Die Aegis von Aggramar wird in den heiligen Hallen des allmächtigen Vaters verwahrt. 
```

Reply:
Hmm... sometimes blizzard does... strange... things to translation/localisation... my one was closer to the english version, but if blizzard wants it this way, keep your version..

Reply:
Roger.